app/controllers/users/phone_confirmation_controller.rb

```diff
@@ -33,11 +33,12 @@ def assign_phone
       old_phone = current_user.phone
       @updating_existing_number = old_phone.present?
       if @updating_existing_number
-        analytics.track_event('User changed and confirmed their phone number')
+        analytics.track_event('User changed their phone number')
         SmsSenderNumberChangeJob.perform_later(old_phone)
```

Comment:
so we continue to notify the user that their MFA number changed, but not their verified IdV number?